### ISSUE TITLE
Doesn't work 

### ISSUE THREAD

**Issue by juxuanu** [flathub/re.sonny.Workbench#9]
```
❯ flatpak run re.sonny.Workbench
/app/bin/workbench: línia 3: /usr/lib/sdk/vala/enable.sh: El fitxer o directori no existeix
```

**Comment by sonnyp** [comment]
Can you try again after

```
flatpak install org.freedesktop.Platform.Extension.vala
```

**Comment by sonnyp** [comment]
Fixed with 43.2 available on Flathub.

**Comment by juxuanu** [comment]
Thanks! 👍
